src/main/kotlin/lotto/domain/LottoPurchaseAmount.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.domain
+
+data class LottoPurchaseAmount(val value: Int) {
+
+    init {
+        require(value % PURCHASE_AMOUNT_UNIT == 0) { "Lotto purchase amount must be a multiple of $PURCHASE_AMOUNT_UNIT" }
+    }
+
+    fun getLottoAmount(): Int = value / PURCHASE_AMOUNT_UNIT
```

Comment:
```suggestion
val lottoAmount: Int
        get() = value / PURCHASE_AMOUNT_UNIT
```
I think getLottoAmount is also a property of LottoPurchaseAmount.
This way feels more expressive to me. What do you think?

Reply:
Can't agree more!!!!!